build.gradle

```diff
@@ -810,7 +810,7 @@ artifactory {
     publish {
         contextUrl = 'https://software.r3.com/artifactory'
         repository {
-            repoKey = 'corda-dependencies-dev'
+            repoKey = System.getenv('CORDA_ARTIFACTORY_REPOKEY') ?: System.getProperty('corda.artifactory.repokey')
```

Comment:
This will break publishing from the command line. We'd need something more like:
```
System.getenv('CORDA_ARTIFACTORY_REPOKEY') ?: 'corda-dependencies-dev`
```

Reply:
good point - updated